Add  to configuration server_processes_manager module (#9189)
aces/Loris: modules/server_processes_manager/README.md

## Configurations

> The following configurations affect the load of the server_processes_manager module:
> 
>  - /opt/Loris-MRI/bin/mri/
> 
>     - This setting displays the Page of the `server_processes_manager` module.
>         
>         - To display the module you need to add the LORIS-MRI code path in the configuration module.

Reviewer: This description is still indicating that "/opt/Loris-MRI/bin/mri/" must be set even though this is a configurable path. I think something more like one of the suggestions below would be sufficient, shifting the focus on the config setting rather than it's potential value

```suggestion
The following configurations affect the load of the server_processes_manager module:

 - `LORIS-MRI code` path of the `Imaging Pipeline` tab in the configuration module must be set
```
```suggestion
The `LORIS-MRI code` path of the `Imaging Pipeline` tab in the configuration module must be set for the `server_processes_manager` module to load.
```

>         - To display the module you need to add `/opt/Loris-MRI/bin/mri/` to the LORIS-MRI code path in configuration module.

Author: To display the module you need to add the LORIS-MRI code path in the configuration module.
Is that ok?

>         - To display the module you need to add `/opt/Loris-MRI/bin/mri/` to the LORIS-MRI code path in configuration module.

Reviewer: That sounds good to me
